target/linux/kirkwood/patches-5.4/110-pogo_e02.patch

```diff
@@ -30,3 +30,39 @@
  			label = "pogo_e02:orange:fault";
  			gpios = <&gpio1 17 GPIO_ACTIVE_LOW>;
  		};
+@@ -95,24 +102,24 @@
+ 	status = "okay";
+ 
+ 	partition@0 {
+-		label = "u-boot";
+-		reg = <0x0000000 0x100000>;
++		label = "uboot";
++		reg = <0x0000000 0xe0000>;
```

Comment:
If we are going to use 0xe0000 in uboot_env, why don't we use just 0x0 here?

Reply:
what do you mean? where you want to use 0x0?

Reply:
I mean, in replace of 0x0000000.

Reply:
that's cosmetic. It is correct either way.

Reply:
Yes, it's cosmetic, but I wanted it to be consistent. I won't mind it if @adrianschmutzler is also okay with it.

Reply:
@bobafetthotmail 

Either no leading zeros for all partitions (i.e. changing to 0x0 here) or the same number of digits (=7) for all. Currently, it's mixed, and that's misleading on a quick look.

Depending on your decision, I can also adjust this during merge.